### PR TITLE
Initial contributing

### PR DESCRIPTION
**Pull request by dragan**
Initial documentation on contributing to the project

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+##  Project Rename
+
+## Project Management
+    - We do little agile, the little a is the indicator. Which means we're not zealots when it comes to our process.
+### Definintions
+
+  - Epic
+  - Story
+    - When creating stories, follow the INVEST principle.
```
Comment (weitenghuang) [inline CONTRIBUTING.md]: Github renders the INVEST definition items in one line instead of bullet points.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      1. Automated Tests
+      2. Documentation
+      3. Styling
+      4. Signed Contributor License Agreement (CLA)?
+      5. Signed off by commit, if an outside contributor.
+
+#### Pipelines
+
+- Triage
+  - New issues land here automatically. They should be dragged to another pipeline as soon as possible or closed.
```
Comment (weitenghuang) [inline CONTRIBUTING.md]: Add space for line break

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    - Goal to get accepted by the Cloud Native Computing Foundation (CNCF) https://www.cncf.io/
+      - https://github.com/cncf/toc/blob/master/process/project_proposals.adoc
+    - Community Repo with all documentation
+      - Use k8s as inspiration: https://github.com/kubernetes/community
+
+##  Project Rename
+
+## Project Management
+    - We do little agile, the little a is the indicator. Which means we're not zealots when it comes to our process.
+### Definintions
```
Comment (weitenghuang) [inline CONTRIBUTING.md]: typo: Definitions

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
+## Open Source
+
+    - MIT
+    - Goal to get accepted by the Cloud Native Computing Foundation (CNCF) https://www.cncf.io/
+      - https://github.com/cncf/toc/blob/master/process/project_proposals.adoc
+    - Community Repo with all documentation
+      - Use k8s as inspiration: https://github.com/kubernetes/community
+
+##  Project Rename
```
Comment (weitenghuang) [inline CONTRIBUTING.md]: extra space in `##  Project`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+- Pending
+  - Issues in this pipeline are well-defined and are the current focus for the team. As issue flow into this pipeline, order them
+    by priority. This is where we'll assign issues to ourselves from. Issues should be worked in the order they are presented in the
+    pipeline. No cherry picking.
+- Current
+  - This pipeline shows the issues we're currently working on in this iteration. Each team member should be working on just one thing
+    at a time. When you start on an issue assigned to you in the Pending pipeline. Drag the issue to this pipeline. When you're ready for
+    an code review, issue a PR. You can issue a PR at any point, it just depends on when collaboration is needed.
+- Done
+  - Once an issue satisfy's what we call done, the PR will be accepted and the issue will be dragged here.
```
Comment (weitenghuang) [inline CONTRIBUTING.md]: typo: satisfies

File: CONTRIBUTING.md (hunk 3 above)
Comment (weitenghuang) [inline CONTRIBUTING.md]: do you use `###` intentionally? or it can be `####` as `Pipelines`, `Labels` etc?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  - Once an issue has been deployed, drag it to this pipeline where it will be automatically closed.
+  - Disclaimer: Reason we do not close an issue after Done, is to allow us to add pipelines if needed in the future.
+
+#### Iteration -> Milestone
+
+- Every iteration will have a milestone
+- We do 5 day iterations and deploy when the issue is done.
+- Every release will have a milestone as well which culminate the stories that we want to ship in that release
+
+#### Labels
```
Comment (weitenghuang) [inline CONTRIBUTING.md]: I will need to learn how we are going to label our issues.
Do you mean having labels like `Type/Epic`, `Type/Story`, ...,`Blocked/Copy`...`Priority/Critical`...etc?

Comment (dragan) [inline CONTRIBUTING.md]: Still figuring this out ...

**Comment by dragan** [comment]
@weitenghuang & @nicklathe I fixed the typos, I'm still working on it, so expect more changes as I finalize the document.

**Comment by dragan** [comment]
@weitenghuang Any objections on the changes?

**Pull request merged**